common/src/main/java/org/opensearch/ml/common/agent/MLAgent.java

```diff
@@ -138,7 +139,7 @@ private void validateMLAgentType(String agentType) {
                 MLAgentType.valueOf(agentType.toUpperCase(Locale.ROOT)); // Use toUpperCase() to allow case-insensitive matching
             } catch (IllegalArgumentException e) {
                 // The typeStr does not match any MLAgentType, so throw a new exception with a clearer message.
-                throw new IllegalArgumentException(agentType + " is not a valid Agent Type");
+                throw new IllegalArgumentException("Invalid Agent Type, Please use one of " + Arrays.toString(MLAgentType.values()));
```

Comment:
From previous security reviews, suggest not returning the valid value list in error message